h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -733,6 +733,12 @@ private void setCommonModelBuilderParams(Model.Parameters params) {
           params._fold_assignment = Model.Parameters.FoldAssignmentScheme.Modulo;
         }
       }
+      if (buildSpec.build_control.balance_classes == true) {
```

Comment:
Is it assumed if `balance_classes` is true then the rest are supplied?

Reply:
Yes, currently if `balance_classes` is true, then we additionally set the following:
```
params._class_sampling_factors = buildSpec.build_control.class_sampling_factors;
params._max_after_balance_size = buildSpec.build_control.max_after_balance_size;
```
Even if `buildSpec.build_control.class_sampling_factors` and/or `buildSpec.build_control.max_after_balance_size` are not set, it should be okay to update `params` because they will take the defaults which are:
```
    public float[] class_sampling_factors;
    public float max_after_balance_size = 5.0f;
```
@navdeep-G Do you agree?

Reply:
Yep, I agree.